src/main/com/nus/duke/data/Searchable.java

```diff
@@ -0,0 +1,6 @@
+package com.nus.duke.data;
+
+public interface Searchable {
```

Comment:
Will it be better to add a JavaDoc here to desribe Searchable? I'm a bit confused about this interface